x/evm/types/interfaces.go

```diff
@@ -67,3 +68,13 @@ type (
 		GetParamSetIfExists(ctx sdk.Context, ps LegacyParams)
 	}
 )
+
+// BankWrapper defines the methods required by the wrapper around
+// the Cosmos SDK x/bank keeper that is used to manage an EVM coin
+// with a configurable value for decimals.
+type BankWrapper interface {
+	BankKeeper
+
+	MintAmountToAccount(ctx context.Context, recipientAddr sdk.AccAddress, amt math.Int) error
+	BurnAmountFromAccount(ctx context.Context, account sdk.AccAddress, amt math.Int) error
```

Comment:
nit: use *big.Int so is cleaner when using it on statedb.go and is consistent with the type used within the EVM
```suggestion
	MintAmountToAccount(ctx context.Context, recipientAddr sdk.AccAddress, amt *big.Int) error
	BurnAmountFromAccount(ctx context.Context, account sdk.AccAddress, amt *big.Int) error
```